### PR TITLE
fix: dont include interval in lag

### PR DESCRIPTION
**Pull request by retrohacker**
## Pre-Submission Checklist

- [ ] Opened an issue discussing these changes before opening the PR
- [x] Ran the linter and tests via `make prepush`
- [x] Included comprehensive and convincing tests for changes

# Changes

> What does this PR do?

Currently the `lag` metric for `cpuUsageThrottle` includes the interval. For example, an interval of `50` firing at the exact right moment will result in a lag of `50` as opposed to `0`. This fixes that.

Also, this PR removes all timeouts after the `cpuUsageThrottle` tests.

### CODE REVIEW

**Comment by retrohacker** [comment]
Hmm, make prepush works for me locally. Removing arrow function.

File: test/plugins/cpuUsageThrottle.test.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ describe('cpuUsageThrottle', function () {
         var opts = { limit: 0, interval: 500 };
         var plugin = cpuUsageThrottle(opts);
         function next (cont) {
-            clearTimeout(plugin._timeout);
+            plugin.close();
```
Comment (DonutEspresso) [inline test/plugins/cpuUsageThrottle.test.js]: Should this just be in an `afterEach` block?

Comment (retrohacker) [reply test/plugins/cpuUsageThrottle.test.js]: Is there a way to do that without maintaining state outside the test?

Comment (DonutEspresso) [reply test/plugins/cpuUsageThrottle.test.js]: Not that I know of - usually just use a `plugin` var within the closure.

**Pull request merged**